bot/exts/evergreen/githubinfo.py

```diff
@@ -91,7 +91,8 @@ async def github_user_info(self, ctx: commands.Context, username: str) -> None:
             )
 
             if user_data["type"] == "User":
-                embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{quote(username, safe='')})")
+                embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{quote_plus(username, safe='')}"
+                                                    f")")
```

Comment:
The indentation here looks messed up.

Reply:
That's a visual indent.

That said, I do guess you could do a hanging indent like this:
```suggestion
                embed.add_field(
                    name="Gists",
                    value=f"[{gists}](https://gist.github.com/{quote_plus(username, safe='')})"
                )
```

Reply:
Fair point, I wanted to pass the linting. I'll push a change shortly.

Reply:
oh yeah I missed that, looks like something an autoformatter would do :eyes:

Reply:
What Bluenix showed is the way it should be done. This is what's specified in our style guide, but I am not sure how strict this project is with enforcing that. In any case, I'm not going to make you push a second commit.

Reply:
Yeah I agree with Mark, I prefer the hanging indent over visual

Reply:
I can push another commit, it's not a problem! :)